src/workloads/execution/UnshardedLookup.yml

```diff
@@ -0,0 +1,70 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  Benchmark a $lookup from one unsharded collection to another in a sharded environment.
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    DocumentCount: 1000
+    BatchSize: &batchSize 1000
+    CollectionCount: 2
+    Document:
+      shardKey: {^RandomInt: {min: 1, max: 1000}}
```

Comment:
Since these are unsharded collections, do we need this 'shardKey' field?

Reply:
No we don't. I was originally creating a sharded collection with a single chunk, but I later realized I could just run with the feature flag on/off. This can be removed.